composer.json

```diff
@@ -19,46 +19,67 @@
-        "typo3/cms-fluid-styled-content": "dev-main as 12.0",
-        "typo3/cms-form": "dev-main as 12.0",
-        "typo3/cms-frontend": "dev-main as 12.0",
-        "typo3/cms-impexp": "dev-main as 12.0",
-        "typo3/cms-indexed-search": "dev-main as 12.0",
-        "typo3/cms-info": "dev-main as 12.0",
-        "typo3/cms-install": "dev-main as 12.0",
-        "typo3/cms-linkvalidator": "dev-main as 12.0",
-        "typo3/cms-lowlevel": "dev-main as 12.0",
-        "typo3/cms-opendocs": "dev-main as 12.0",
-        "typo3/cms-recycler": "dev-main as 12.0",
-        "typo3/cms-redirects": "dev-main as 12.0",
-        "typo3/cms-reports": "dev-main as 12.0",
-        "typo3/cms-rte-ckeditor": "dev-main as 12.0",
-        "typo3/cms-scheduler": "dev-main as 12.0",
-        "typo3/cms-seo": "dev-main as 12.0",
-        "typo3/cms-setup": "dev-main as 12.0",
-        "typo3/cms-sys-note": "dev-main as 12.0",
-        "typo3/cms-t3editor": "dev-main as 12.0",
-        "typo3/cms-tstemplate": "dev-main as 12.0",
-        "typo3/cms-viewpage": "dev-main as 12.0",
-        "typo3/cms-workspaces": "dev-main as 12.0",
-        "t3docs/examples": "dev-main"
+        "typo3/cms-adminpanel": "dev-main",
+        "typo3/cms-backend": "dev-main",
+        "typo3/cms-belog": "dev-main",
+        "typo3/cms-beuser": "dev-main",
+        "typo3/cms-dashboard": "dev-main",
+        "typo3/cms-extbase": "dev-main",
+        "typo3/cms-extensionmanager": "dev-main",
+        "typo3/cms-felogin": "dev-main",
+        "typo3/cms-filelist": "dev-main",
+        "typo3/cms-filemetadata": "dev-main",
+        "typo3/cms-fluid": "dev-main",
+        "typo3/cms-fluid-styled-content": "dev-main",
+        "typo3/cms-form": "dev-main",
+        "typo3/cms-frontend": "dev-main",
+        "typo3/cms-impexp": "dev-main",
+        "typo3/cms-indexed-search": "dev-main",
+        "typo3/cms-info": "dev-main",
+        "typo3/cms-install": "dev-main",
+        "typo3/cms-linkvalidator": "dev-main",
+        "typo3/cms-lowlevel": "dev-main",
+        "typo3/cms-opendocs": "dev-main",
+        "typo3/cms-recycler": "dev-main",
+        "typo3/cms-redirects": "dev-main",
+        "typo3/cms-reports": "dev-main",
+        "typo3/cms-rte-ckeditor": "dev-main",
+        "typo3/cms-scheduler": "dev-main",
+        "typo3/cms-seo": "dev-main",
+        "typo3/cms-setup": "dev-main",
+        "typo3/cms-sys-note": "dev-main",
+        "typo3/cms-t3editor": "dev-main",
+        "typo3/cms-tstemplate": "dev-main",
+        "typo3/cms-viewpage": "dev-main",
+        "typo3/cms-workspaces": "dev-main",
+        "t3docs/examples": "dev-main",
+        "typo3/coding-standards": "^0.7.1"
     },
     "scripts": {
+        "check": [
+            "@check:php",
+            "@check:rst"
+        ],
+        "check:php": [
+            "@check:php:lint",
+            "@check:php:cs"
+        ],
+        "check:php:cs": "Build/Scripts/runTests.sh -n -s cgl",
```

Comment:
Defining composer scripts dispatching to runTests.sh
is kind of a `paradoxon`.

Using `runTests.sh` should eleminate the need to have
the correct php and/or composer version available on
the host to invoke something.

TBH requiring php/composer to invoke the runTests.sh
does not make much sense in my eyes.

Just a remark.

Reply:
This is not a requirement but just an offer. I cannot remember the commands you need to run and in wich order and combination for purpose. I could also introduce a make scritp but most ppl know how to work with composer scripts and have composer anyway.

Reply:
Guess, a discussion for another day. Closing this.